lib/app/migrate.ts

```diff
@@ -12,47 +12,48 @@ import { downloadProject } from '@hubspot/local-dev-lib/api/projects';
 
 import { confirmPrompt, inputPrompt, listPrompt } from '../prompts/promptUtils';
 import { uiAccountDescription, uiCommandReference, uiLine } from '../ui';
-import { i18n } from '../lang';
-import { ensureProjectExists } from '../projects';
+import { ensureProjectExists, LoadedProjectConfig } from '../projects';
 import SpinniesManager from '../ui/SpinniesManager';
 import { DEFAULT_POLLING_STATUS_LOOKUP, poll } from '../polling';
 import { MigrateAppOptions } from '../../types/Yargs';
 import {
   checkMigrationStatusV2,
   continueMigration,
   initializeMigration,
+  isMigrationStatus,
   listAppsForMigration,
   MigrationApp,
   MigrationStatus,
 } from '../../api/migrate';
+import fs from 'fs';
+import { lib } from '../../lang/en';
 
 function getUnmigratableReason(reasonCode: string): string {
   switch (reasonCode) {
     case UNMIGRATABLE_REASONS.UP_TO_DATE:
-      return i18n(
-        'commands.project.subcommands.migrateApp.unmigratableReasons.upToDate'
-      );
+      return lib.migrate.errors.unmigratableReasons.upToDate;
```

Comment:
I was going a bunch of i18n key shuffling, so I figured I might as well onboard these to the new object for the adding safety of compile time checking that the key exists.